src/main/java/com/hillel/vynnyk/homeworks/projectPersonSpring/entity/Account.java

```diff
@@ -0,0 +1,21 @@
+package com.hillel.vynnyk.homeworks.projectPersonSpring.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Table(name = "accounts")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Account extends BaseEntity {
+    private String uid;
+    private String iban;
+    private float balance;
+    private Integer person_id;
```

Comment:
```suggestion
    private Integer personId;
```
має автоматично мапитись на person_id в БД